Project/OOP/Cylinder

```diff
@@ -1 +1,30 @@
-
+class Cylinder:
+    @staticmethod
+    def make_area(d, h):
+        circle = pi * d ** 2 / 4
+        side = pi * d * h
+        return round(circle*2 + side, 2)
+    
+    def __init__(self, diameter, high):
+        self.dia = diameter
+        self.high = high
+        self.area = self.make_area(self.dia,self.high)
+    
+    def __setattr__(self, name, value):
+        if name == 'dia' or name == 'high':
+            self.__dict__[name] = value
+            self.area = self.make_area(self.dia,self.high)
+        elif name == 'area':
+            print('Нелья присваивать значение площади поверхности')
```

Comment:
an exception would be more appropriate here ( if you're using the class for development